src/Authoring/WinRT.SourceGenerator/WinRTRules.cs

```diff
@@ -238,5 +238,12 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
             CsWinRTDiagnosticStrings.ClassImplementsOldProjection_Brief,
             CsWinRTDiagnosticStrings.ClassOldProjectionMultipleInstances_Text,
             false);
+
+        public static DiagnosticDescriptor NonEmptyCollectionExpressionTargetingNonBuilderInterfaceType = MakeRule(
```

Comment:
1032?

Reply:
Mmh wait, what other PR will be using 1031?